test/Tester/PSClientTests/PSClientTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Orleans;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using OrleansPSUtils;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+using UnitTests.Tester;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Tester
+{
+    public class PowershellHostFixture : DefaultClusterFixture
```

Comment:
This is outdated @jdom. The current code in the PR already has the change you suggested and inherit from `BaseTestClusterFixture`